src/app/frontend/deploy/createsecret.html

```diff
@@ -16,28 +16,28 @@
 
 <md-dialog aria-label="Create a new secret" layout="column" class="kd-create-secret-dialog">
   <md-dialog-content layout-padding>
-    <h4 class="md-title">Create a new image pull secret</h4>
-    <div>The new secret will be added to the cluster.</div>
+    <h4 class="md-title">{{ctrl.messages.MSG_IMAGE_PULL_SECRET_CREATE_DIALOG_TITLE}}</h4>
```

Comment:
what about shortening the messages variable:

ctrl.m.MSG_IMAGE_PULL_SECRET_CREATE_DIALOG_TITLE


ctrl.i18n.MSG_IMAGE_PULL_SECRET_CREATE_DIALOG_TITLE